### ISSUE TITLE
[DotNet Core Global] pwsh fails when user name contains spaces

### ISSUE THREAD

**Issue by eobeda** [PowerShell/PowerShell#10184]
# Steps to reproduce

After installing as a global DotNet command,
``` dotnet tool install --global PowerShell ```

execute  a simple command
``` pwsh -c Get-Date ```

# Expected behavior

Display today's date.

# Actual behavior

```
 C:\Users\ed obeda>dotnet tool install --global PowerShell

You can invoke the tool using the following command: pwsh
Tool 'powershell' (version '6.2.2') was successfully installed. 
```
Note the space in my user name. Changing execution location ie ```c:\``` does not resolve the issue.
``` 
C:\Users\ed obeda>pwsh -c Get-Date

Could not execute because the specified command or file was not found.
Possible reasons for this include:
  * You misspelled a built-in dotnet command.
  * You intended to execute a .NET Core program, but dotnet-C:\Users\ed does not exist.
  * You intended to run a global tool, but a dotnet-prefixed executable with this name could not be found on the PATH.

```

# Environment data

Windows 10 Enterprise. Provide the output of $PSVersionTable  fails with the same error.

**Comment by travisterrell** [comment]
I believe I am still having this issue, despite using latest version. I cannot speak  Any advice on what to do?
![image](https://user-images.githubusercontent.com/14238582/70162750-ce8b8b80-1683-11ea-8e57-8d8db0e285a6.png)

**Comment by vexx32** [comment]
@travisterrell this change wasn't included in a release until 7.0-preview4. I'm not sure if preview releases are packaged up for use as dotnet global tools or not; either there's a flag for allowing install of prereleases of tools, or you'll need to wait for the 7.0 release, which is likely sometime in January. 🙂

**Comment by travisterrell** [comment]
@vexx32 Ah, cool deal! I'll build it and see if things work. Thanks for the info.
